mrfCommon/src/flashiocsh.cpp

```diff
@@ -104,7 +104,7 @@ void flashread(const char *name, int addrraw, int countraw, const char *outfile)
         if(outfile) {
             fstrm.open(outfile, std::ios_base::out|std::ios_base::binary);
             if(fstrm.fail())
-                throw std::runtime_error("Unable to open output file");
+                throw std::runtime_error("Unable to open input file");
```

Comment:
Is that change right ?

Reply:
Thanks for finding out this mistake. This correction should be in the flashwrite function. I will make a correction on this commit.